.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+
+python:
+    - 2.7
+
+env: DISPLAY=:99
+
+before_install:
+    - sudo apt-get update
+    - sh -e /etc/init.d/xvfb start
+
+install:
+    - sudo apt-get install flashplugin-nonfree
+    - pip install mozdownload pep8
+    - python setup.py develop
```

Comment:
I wonder if we should move the call to setup.py down and run pep8 before. That would save us some seconds and an unnecessary installation of the marionette package and all deps.